tapiriik/services/Aerobia/aerobia.py

```diff
@@ -0,0 +1,340 @@
+        55 : ActivityType.Other, #roller sport
+        77 : ActivityType.Running, #tredmill
+        66 : ActivityType.Other, #roller skiing
+        7 : ActivityType.Other, #rollers
+        58 : ActivityType.Other, #nordic walking
+        10 : ActivityType.Other, #snowboarding
+        16 : ActivityType.Other, #walking sport
+        18 : ActivityType.Other, #orienting
+        38 : ActivityType.Other, #OTHER
+        61 : ActivityType.Other, #WATER AEROBICS
+        79 : ActivityType.Other, #ACROBATICS
+        23 : ActivityType.Other, #AEROBICS
+        26 : ActivityType.Other, #BOX
+        84 : ActivityType.Other, #CYCLOCROSS
+        24 : ActivityType.Other, #BADMINTON
+        52 : ActivityType.Other, #VOLLEYBALL
+        50 : ActivityType.Other, #MARTIAL ARTS
+        49 : ActivityType.Other, #HANDBALL
+        48 : ActivityType.Other, #GYMNASTICS
+        4 : ActivityType.Other, #GOLF
+        36 : ActivityType.Other, #SCUBA DIVING
+        85 : ActivityType.Other, #DUATHLON
+        69 : ActivityType.Other, #DELTAPLAN
+        47 : ActivityType.Other, #YOGA
+        45 : ActivityType.Other, #KITEBOARDING
+        80 : ActivityType.Other, #KERLING
+        62 : ActivityType.Other, #HORSE RIDING
+        71 : ActivityType.Other, #СROSSFIT
+        64 : ActivityType.Other, #CIRCLE WORKOUT
+        78 : ActivityType.Other, #MOTORSPORT
+        44 : ActivityType.Other, #ММА
+        70 : ActivityType.Other, #PARAPLANE
+        35 : ActivityType.Other, #PILATES
+        20 : ActivityType.Other, #POLO
+        33 : ActivityType.Other, #RUGBY
+        60 : ActivityType.Other, #FISHING
+        67 : ActivityType.Other, #SCOOTER
+        15 : ActivityType.Other, #WINDSURFING
+        42 : ActivityType.Other, #SQUASH
+        41 : ActivityType.Other, #SKATEBOARD
+        75 : ActivityType.Other, #STEPPER
+        29 : ActivityType.Other, #DANCING
+        40 : ActivityType.Other, #TENNIS
+        37 : ActivityType.Other, #TABLE TENNIS
+        81 : ActivityType.Other, #OUTDOOR FITNESS
+        31 : ActivityType.Other, #FOOTBALL
+        59 : ActivityType.Other, #FENCING
+        39 : ActivityType.Other, #FIGURE SKATING
+        34 : ActivityType.Other, #HOCKEY
+        82 : ActivityType.Other, #CHESS
+
+        68 : ActivityType.Other
+    }
+
+    SupportedActivities = list(_activityMappings.keys())
+
+    SupportsHR = SupportsCadence = True
+
+    SupportsActivityDeletion = True
+
+    _urlRoot = "http://aerobia.ru/"
+    _apiRoot = "http://aerobia.ru/api/"
+    _loginUrlRoot = _apiRoot + "sign_in"
+    _workoutsUrl = _apiRoot + "workouts"
+    _workoutUrl = _apiRoot + "workouts/{id}.json"
+    _uploadsUrl = _apiRoot + "uploads.json"
+
+    def _patch_user_agent(self):
+        from tapiriik.requests_lib import patch_requests_user_agent
+        # Without user-agent patch aerobia requests doesn't work
+        patch_requests_user_agent('Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; rv:11.0) like Gecko')
```

Comment:
This would be better as a requests session object (e.g. what happens in Garmin Connect) - that way it doesn't change the behavior of other services using requests.